content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,185 @@
+decisions around the GNU Radio project is the fact that we decided this behind
+closed doors within the leadership team. For example, we did not publish a GREP
+ahead of time for a public discussion.
+
+After extensive discussions, we agreed that without actually having tried it,
+the decision for a chat service would almost exclusively be a battle of
+opinions. Having all reasonable opinions among us, we decided to forgo
+fragmenting the community and instead conducted a closed testing of the most
+promising candidates.
+
+## What happens to the GNU Radio Slack?
+
+We will turn off Slack end of August 2020.
+The main development, documentation, and variety of other discussions regarding
+the project will be held on our
+Matrix instance going forward, in order to preserve them in history.
+We are paying for a one-time unlocking of the Slack history, and will make
+them available in some form to all participants of the GNU Radio chat.
+
+Many people from the GNU Radio leadership will be leaving the Slack server, and
+won't be reachable there going forward, even before August.
+
+We could keep Slack running, since it's free, but we decided against it. This is
+for multiple reasons:
+- Even if administering the Slack instance is very little work, it is still work
+  and we don't want to have yet another task.
+- All GNU Radio leadership will move off of the Slack channel, so calling it a
+  GNU Radio chat service is a stretch anyway at that point.
+- Closing the service is the only way to guarantee that everyone on the GNU Radio
+  Slack definitely hears about the deprecation of our Slack.
+- We don't want to risk any GPDR issues by being responsible for other people's
+  data.
+
+## The new setup, and yes, we're still tuning it
+
+As of April 2020, we have rented a hosted Matrix setup at Swiss hosting company
+[unglei.ch](https://ungleich.ch/u/products/hosted-matrix-chat/). Our custom
+entry point [chat.gnuradio.org](https://chat.gnuradio.org) points to their
+server, but unlike Slack, we can pack up our chat and move it elsewhere if we're
+unhappy with the hosting service provided. The hosting service was chosen based
+on an internet research and a comparison of services provided, and cost. We have
+no other affiliations with the hosting company.
+
+We only recently set this up, and we probably want to tune this quite a bit. For
+example, we don't have any integrations set up such as the GitHub ticker, and
+we're not even sure if we need it. We're still experimenting with the IRC
+gateway, which might be most confusing right now, as we haven't fully set it up
+like we want it to be, and we're also bridging over into Slack.
+
+
+Join us on Matrix and give us your feedback!
+
+## Is that it? When's the next Chat server move?
+
+Hopefully, we'll never move chat services again. As mentioned before, the existing
+chat service on [chat.gnuradio.org](https://chat.gnuradio.org) is still
```

Comment:
I thought it is [matrix.gnuradio.org](matrix.gnuradio.org). Given the fact that  [chat.gnuradio.org](chat.gnuradio.org) ends up on an landing page for a newly installed nginx server, I think this link should be updated.

Reply:
Tomorrow this will be switched to chat.gnuradio.org :)